src/styles/blocks/card/_card.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../utils/variables";
+@import "../../utils/mixins";
+
+.product-list {
+  display: flex;
+  justify-content: center;
+}
+
+.product-list__container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 0 72px;
+  width: $section-width;
+
+}
+
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-right: $card-margin;
+
+  border: 1px solid $primary-brdr;
+  border-radius: 5px;
+
+  @include default-text;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin: 32px 15px 40px;
+  }
+
+  &__name {
+    margin: 0 16px 4px;
+    color: $primary-text;
+  }
+
+  &__pid {
+    margin: 0 auto 16px 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-text;
+  }
+
+  &__body {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+
+    padding: 0 16px 24px;
+  }
+
+  &__reviews {
+    margin-left: auto;
+    align-self: flex-end;
+    @include small-text;
+    color: $primary-text;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    padding: 0 16px 16px;
+  }
+
+  &__price-price {
+    font-weight: 300;
+    color: $secondary-text;
+  }
+
+  &__price-numbers {
+    margin: 0 0 0 auto;
+
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $primary-text;
+  }
+
+  &__btn {
+    display: block;
+    padding: 11px 69px;
+    margin: 0 16px 16px;
+
+    background-color: $secondary-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    color: $btn-secondary;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    transition: 0.3s;
+  }
+
+  &__btn:hover {
+    background-color: $btn-secondary;
+    color: $secondary-color;
+    border: 1px solid $secondary-color;
+  }
+
+}
+
+.card:nth-last-child(-n + 4) {
+  margin-top: 46px;
+  order: 2;
+}
```

Comment:
Why do you add margin-top to last 4 elements? You have to add them margin-bottom 0px, as I understand the task.